frontend/src/Store/Selectors/createProfileInUseSelector.js

```diff
@@ -6,12 +6,17 @@ function createProfileInUseSelector(profileProp) {
   return createSelector(
     (state, { id }) => id,
     createAllSeriesSelector(),
-    (id, series) => {
+    (state) => state.settings.importLists.items,
+    (id, series, lists) => {
       if (!id) {
         return false;
       }
 
-      return _.some(series, { [profileProp]: id });
+      if (_.some(series, { [profileProp]: id }) || _.some(lists, { [profileProp]: id })) {
```

Comment:
Good time to refactor away from lodash and no need to avoid returning directly.

`return series.some((s) => s[profileProp] === id) || lists.some((list) => list[profileProp] === id);`